src/mumble/Messages.cpp

```diff
@@ -721,9 +722,22 @@ void MainWindow::msgTextMessage(const MumbleProto::TextMessage &msg) {
 		privateMessage = true;
 	}
 
+	//If NoScope or NoAuthor is selected generate a new string to pass to TTS
+	if (g.s.bTTSNoScope || g.s.bTTSNoAuthor) {
+		if (g.s.bTTSNoScope && g.s.bTTSNoAuthor){
+			shortTTS += tr("%2%1: %3").arg(QString()).arg(QString()).arg(u8(msg.message()));
+		}else if (g.s.bTTSNoAuthor){
+			shortTTS += tr("%2%1: %3").arg(QString()).arg(target).arg(u8(msg.message()));
+		}else if (g.s.bTTSNoScope){
```

Comment:
There should be a space after the closing bracket and before the opening bracket.